src/styles/globals.scss

```diff
@@ -86,4 +87,4 @@ h2 {
   line-height: 1.5;
```

Comment:
it looks like option+shift+F is needed to prettify this file too ☺️

Reply:
as u wish, my mistress